internal/lib/container_server_test.go

```diff
@@ -156,6 +156,19 @@ var _ = t.Describe("ContainerServer", func() {
 			Expect(err).To(BeNil())
 		})
 
+		It("should succeed load infraContainer", func() {
+			// Given
+			createDummyState()
+			mockDirs(testManifest)
+
+			// When
+			_, err := sut.LoadSandbox(context.Background(), "id")
+
+			// Then
+			Expect(err).To(BeNil())
+			Expect(sut.GetInfraContainer(sandboxID)).NotTo(BeNil())
```

Comment:
**To the reviewer:**

Updated for CRI-O 1.25 to resolve the following function signature difference:

```console
# github.com/cri-o/cri-o/internal/lib_test [github.com/cri-o/cri-o/internal/lib.test]
./container_server_test.go:169:55: too many arguments in call to sut.GetInfraContainer
	have ("context".Context, string)
	want (string)
```